src/pypowsybl_jupyter/selectcontext.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2024, RTE (http://www.rte-france.com)
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+
+from collections import OrderedDict
+import pandas as pd
+from pypowsybl.network import Network
+
+class SelectContext:
+
+    def __init__(self, network:Network = None, vl_id : str = None, use_name:bool = True):
+        self.network = network
+        self.use_name = use_name
+        self.display_attribute = 'name' if use_name else 'id'
+
+        self.vls = network.get_voltage_levels(attributes=['name'])
```

Comment:
Just thinking about what could be done to display only specific kind of voltage levels, for a coming PR! To do something similar to the javaFX viewer where you could choose to display only voltage levels with specific equipments. That would mean having here a parameter, either a function (a voltage level provider) or a filter. We'll speak about it later!